stingray/io.py

```diff
@@ -2,6 +2,7 @@
                         print_function)
 import numpy as np
 import logging
+import pickle
```

Comment:
Following the astropy guidelines, shouldn't we be using `six` to do this? See here: http://docs.astropy.org/en/stable/development/codeguide.html

A cursory glance at `six` makes me think that `six.moves` might be the right thing to use here.